core/sail/fts/solr/pom.xml

```diff
@@ -42,6 +43,15 @@
 			</exclusions>
 		</dependency>
 
+		<!-- jdk.tools -->
+		<dependency>
+			<groupId>jdk.tools</groupId>
+			<artifactId>jdk.tools</artifactId>
+			<version>1.8</version>
+			<scope>system</scope>
+			<systemPath>${JAVA_HOME}/lib/tools.jar</systemPath>
```

Comment:
It will work if `$JAVA_HOME` is correctly configured (pointing to something like `/Library/Java/JavaVirtualMachines/jdk<version>.jdk/Contents/Home`). But a standard install of the JDK doesn't set that var, I don't think.